src/components/Records/Search/Input/StringSearch.vue

```diff
@@ -1,36 +1,60 @@
 <template>
   <form
-    class="stringSearchBar d-flex flex-row align-center mt-1  mr-2 ml-2"
+    class=" d-flex flex-row align-center align-content-center pt-1 mr-1 mr-lg-1 ml-1"
     @submit.prevent="searchString()"
   >
     <v-text-field
       v-model="searchTerm"
       solo
       single-line
       clearable
-      placeholder="Can't find what you're looking for? Search through all data."
+      dense
+      full-width
+      :class="$vuetify.breakpoint.lgAndDown?'v-input':'v-input-lg-up'"
+      :height="responsiveHeightTextBox"
+      :placeholder="placeHolder"
     />
     <v-btn
       color="primary"
       outlined
-      height="52px"
-      class="mt-1 ml-2"
+      :class="responsiveHeight"
+      class="mt-1 mt-lg-1 ml-2"
       @click="searchString()"
     >
       <v-icon>search</v-icon>
-      <span>Search</span>
+      <span class="button-text-size">Search</span>
     </v-btn>
   </form>
 </template>
 
 <script>
 export default {
   name: "StringSearch",
+  props: {
+    placeHolder: {default: "", type: String}
```

Comment:
use placeholder instead of placeHolder so you can have a single word.

Reply:
I disagree. please read this link. this is vue convention. 
https://vuejs.org/v2/guide/components-props.html